cmd/wfx/cmd/root/root.go

```diff
@@ -0,0 +1,151 @@
+package root
+
+/*
+ * SPDX-FileCopyrightText: 2024 Siemens AG
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Author: Michael Adler <michael.adler@siemens.com>
+ */
+
+import (
+	"fmt"
+	"os"
+	"os/signal"
+	"os/user"
+	"sync"
+	"syscall"
+	"time"
+
+	"github.com/Southclaws/fault"
+	"github.com/rs/zerolog/log"
+	"github.com/siemens/wfx/cmd/wfx/cmd/config"
+	"github.com/siemens/wfx/cmd/wfx/metadata"
+	"github.com/siemens/wfx/internal/man"
+	"github.com/siemens/wfx/persistence"
+	"github.com/spf13/cobra"
+	"go.uber.org/automaxprocs/maxprocs"
+)
+
+func NewCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "wfx",
+		Short: "wfx server",
+		Long: `This API allows to create and execute reusable workflows for clients.
```

Comment:
Hm, this is not an API in the strictest sense of the wording. Also the "[...] state machine running in the storage" sentence is a bit imprecise.

Reply:
I've pushed an update improving the description